src/python/ensembl/io/genomio/schemas/__init__.py

```diff
@@ -12,7 +12,4 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""TODO"""
-
-from .json_schema_validator import *
-
+"""Schemas module."""
```

Comment:
Missing the import area:
```
from .json_schema_validator import *
```
Can be left until we restructure the folders, but this might not be necessary.